src/CacheEvents.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gregurco\Bundle\GuzzleBundleCachePlugin;
```

Comment:
Why you put this class in `src` folder and not in `src/Event`? I see you already created this folder...

Reply:
Because of how symfony deals with this kind of classes: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/KernelEvents.php (root of the component) so it is a common practice to put bundle related events at the root of it. Can move it if you prefer

Reply:
It's ok. I didn't know about this practice. Thank you for information :slightly_smiling_face: